newIDE/app/src/Utils/GDevelopServices/Authentication.js

```diff
@@ -59,22 +60,14 @@ export type RegisterForm = {|
   getNewsletterEmail: boolean,
 |};
 
-export type AdditionalUserInfoForm = {|
-  gdevelopUsage?: string,
-  teamOrCompanySize?: string,
-  companyName?: string,
-  creationExperience?: string,
-  creationGoal?: string,
-  hearFrom?: string,
-|};
-
 export type EditForm = {|
-  username: string,
-  description: string,
-  getGameStatsEmail: boolean,
-  getNewsletterEmail: boolean,
-  donateLink: string,
-  communityLinks: CommunityLinks,
+  username?: string,
```

Comment:
I'm wondering if we should keep this type strict (all values) because it represents the editForm in the profile, and use another type / function for when sending the survey? unsure about that